caddyhttp/httpserver/context.go

```diff
@@ -26,6 +26,7 @@ import (
 // Context is the context with which Caddy templates are executed.
 type Context struct {
 	Root http.FileSystem
+	Res  http.ResponseWriter
```

Comment:
However, by exposing this, it also lets people call Write and WriteHeader on the ResponseWriter from a template. I'm not sure if this is desirable, in fact, probably not. What if we just expose the Header field instead of the whole ResponseWriter?